packages/app/src/cli/services/bulk-operations/execute-bulk-operation.ts

```diff
@@ -105,7 +91,8 @@ export async function executeBulkOperation(input: ExecuteBulkOperationInput): Pr
         await renderBulkOperationResult(operation, outputFile)
       }
     } else {
-      await renderBulkOperationResult(createdOperation, outputFile)
+      const operation = await quickWatchBulkOperation(adminSession, createdOperation.id)
```

Comment:
Curious why you opted to make this a new function. Other options might have been:

- do a fetch inline right here
- make it an option on the normal `watchBulkOperation` (e.g. a `maxWatchTime` parameter)